packages/gestalt/src/contexts/Theme.js

```diff
@@ -0,0 +1,132 @@
+// @flow strict
+import * as React from 'react';
+import PropTypes from 'prop-types';
+
+export type ColorScheme = 'light' | 'dark' | 'userPref';
+
+export const ColorSchemePropType: React$PropType$Primitive<ColorScheme> = PropTypes.oneOf(
+  ['light', 'dark', 'userPref']
+);
+
+type Theme = {|
+  name: string,
+  colorRed0: string,
+  colorRed100: string,
+  colorGray0: string,
+  colorGray50: string,
+  colorGray100: string,
+  colorGray200: string,
+  colorGray300: string,
+  colorGray400: string,
+|};
+
+type Props = {|
+  children: React.Node,
+  colorScheme?: ColorScheme,
+|};
+
+const lightModeTheme = {
+  name: 'lightMode',
+  colorRed0: '#ff5247',
+  colorRed100: '#e60023',
+  colorGray0: '#fff',
+  colorGray50: '#fff',
+  colorGray100: '#efefef',
+  colorGray200: '#767676',
+  colorGray300: '#111',
+  colorGray400: '#000',
+};
+
+const darkModeTheme = {
+  name: 'darkMode',
+  colorRed0: '#e60023',
```

Comment:
@jennyscript would there be a way to use CSS modules in this case? And swap a classname on the depending on whether it’s light or dark theme?

Reply:
I wanted to pass the colors down in context for example to support updating fill in some of our svgs. It seemed good to only set them in one place if they need to be in JS anyway. Is it possible to read the colors out of CSS modules to reverse where they're stored?

Reply:
I updated `GroupAvatar` to use the color from context as an example of what I meant